packages/design-system/src/components/_OcTextInputPassword/_OcTextInputPassword.vue

```diff
@@ -51,23 +50,27 @@
   <portal v-if="showPasswordPolicyInformation" to="app.design-system.password-policy">
     <div class="oc-text-small oc-flex oc-flex-column">
       <span v-text="$gettext('Please enter a password that meets the following criteria:')" />
-      <div
-        v-for="(testedRule, index) in testedPasswordPolicy.rules"
-        :key="index"
-        class="oc-flex oc-flex-middle"
-      >
-        <oc-icon
-          size="small"
-          :name="testedRule.verified ? 'check' : 'close'"
-          :variation="testedRule.verified ? 'success' : 'danger'"
-        />
-        <span
-          :class="[
-            { 'oc-text-input-success': testedRule.verified },
-            { 'oc-text-input-danger': !testedRule.verified }
-          ]"
-          v-text="getPasswordPolicyRuleMessage(testedRule)"
-        ></span>
+      <div class="oc-flex oc-text-input-password-policy-rule-wrapper">
```

Comment:
```suggestion
      <div class="oc-text-input-password-policy-rule-wrapper">
```
You already gave the wrapper class `display: flex;` so the `oc-flex` should not be needed.

Reply:
hm this is quite needed as we need to apply     
```scss
flex-direction: row;
flex-wrap: wrap;
``` 

I removed the wrapping flexbox